source/_components/binary_sensor.xiaomi_gw.markdown

```diff
@@ -0,0 +1,162 @@
+date: 2017-07-21 16:34
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: xiaomi.png
+ha_category: binary_sensor
+ha_release: 0.50
+---
+
+To get your Xiaomi binary sensors working with Home Assistant, follow the instructions for the general [Xiaomi Gateway component](https://home-assistant.io/components/xiaomi_gw)
+
+### Type of sensors supported:
+- Motion
+- Door / Window
+- Smoke
+- Gas
+- Xiaomi Wireless Button
+- Xiaomi Cube
+
+### Some automation examples to get you started:
+- Motion
+```yaml
+    # Trigger for motion sensor
+
+    - alias: If there is motion and its dark turn on the gateway light
+      trigger:
+        platform: state
+        entity_id: binary_sensor.motion_sensor_158d000xxxxxc2
+        from: 'off'
+        to: 'on'
+      condition:
+        condition: numeric_state
+        entity_id: sensor.illumination_34ce00xxxx11
+        below: 300
+      action:
+        service: light.turn_on
+        entity_id: light.gateway_light_34ce00xxxx11
+        data:
+          brightness: 5
+
+    - alias: If there no motion for 5 minutes turn off the gateway light
+      trigger:
+        platform: state
+        entity_id: binary_sensor.motion_sensor_158d000xxxxxc2
+        from: 'on'
+        to: 'off'
+        for:
+          minutes: 5
+      action:
+        service: light.turn_off
+        entity_id: light.gateway_light_34ce00xxxx11
+  ```
+  
+- Door / Window
+ ```yaml
+    # Trigger for door window sensor
+
+    - alias: If the window is open turn off the radiator
+      trigger:
+        platform: state
+        entity_id: binary_sensor.door_window_sensor_158d000xxxxxc2
+        from: 'off'
+        to: 'on'
+      action:
+        service: climate.set_operation_mode
+        entity_id: climate.livingroom
+        data:
+          operation_mode: 'Off'
+
+    - alias: If the window is closed for 5 minutes turn on the radiator again
+      trigger:
+        platform: state
+        entity_id: binary_sensor.door_window_sensor_158d000xxxxxc2
+        from: 'on'
+        to: 'off'
+        for:
+          minutes: 5
+      action:
+        service: climate.set_operation_mode
+        entity_id: climate.livingroom
+        data:
+          operation_mode: 'Smart schedule'
+  ```
+
+  ```yaml
+    # Trigger for door window sensor
+
+    - alias: Send notification on fire alarm
+      trigger:
+        platform: state
+        entity_id: binary_sensor.smoke_sensor_158d0001574899
+        from: 'off'
+        to: 'on'
+      action:
+        - service: notify.html5
+          data:
+            title: Fire alarm!
+            message: Fire/Smoke detected!
+        - service: xiaomi.play_ringtone
+          data:
+            gw_mac: xxxxxxxxxxxx
+            ringtone_id: 2
+            ringtone_vol: 100
+  ```
+- Smoke
+- Gas
+- Xiaomi Wireless Button (available events are `single`, `double`, `hold`, `long_click_press` and `long_click_release`).
+ ```yaml
```

Comment:
The new/square version of the Xiaomi Wireless Button just supports `single` and `double` click. Furthermore the space between two clicks to generate a double click must be quite large now.